src/tiled/mainwindow.h

```diff
@@ -253,6 +253,8 @@ public slots:
     DocumentManager *mDocumentManager;
     ToolManager *mToolManager;
     TileStampManager *mTileStampManager;
+private slots:
+    void setUseDarkTheme(bool useDarkTheme);
```

Comment:
Please just place it together with the rest of the slots, for example after the `onCollisionEditorClosed` slot. The fact that those slots are currently marked as `public slots` can be ignored, that's something I want to fix eventually.